analysis_options.yaml

```diff
@@ -17,7 +17,6 @@ linter:
     cancel_subscriptions: true
     close_sinks: true
     directives_ordering: true
-    package_api_docs: true
```

Comment:
What is the reason to remove this lint rule?

Reply:
Removed in the latest versions of Flutter. Maybe only on beta for now.

Reply:
> NOTE: This lint is deprecated because it is has not been fully functional since at least Dart 2.0. Remove all inclusions of this lint from your analysis options.

Oh that's interesting.

Reply:
Yeah, bit strange. Always worked well for me.